src/Piece/PieceForm/PieceFormContainer.test.js

```diff
@@ -26,6 +30,16 @@ import {
 import { usePieceQuickReceiving } from '../hooks';
 import { PieceFormContainer } from './PieceFormContainer';
 
+const queryClient = new QueryClient();
+
+const wrapper = ({ children }) => (
+  <MemoryRouter>
+    <QueryClientProvider client={queryClient}>
+      {children}
+    </QueryClientProvider>
+  </MemoryRouter>
+);
+
```

Comment:
this part of the code is duplicated across the module, I suggest moving it to some helpers.